docs/diagnostics/index.rst

```diff
@@ -4,14 +4,8 @@
 Plasma diagnostics (`plasmapy.diagnostics`)
 *******************************************
 
-.. currentmodule:: plasmapy.diagnostics.langmuir
+.. toctree::
```

Comment:
```suggestion
.. automodapi:: plasmapy.diagnostics
    :no-heading:

.. toctree::
```

This will add the docstring from `plasmapy/diagnostics/__init__.py`.